tools/clearShm/main.cpp

```diff
@@ -213,6 +259,10 @@ int main(int argc, char** argv)
     semDoit(BrmKeys.KEYRANGE_VSS_BASE,        "VSS        ");
     semDoit(BrmKeys.KEYRANGE_EXTENTMAP_INDEX_BASE, "EXTMAP_INDX");
     semDoit(BrmKeys.MST_SYSVKEY,              "MST        ");
+    // We have only one segment, and grow it on request.
+    // FIXME: outline to constant.
+    semDoit("InfiniDB-shm-shared_managed_segment_00", "EXTENT_MAP_RB_TREE");
+    shmDoit("InfiniDB-shm-shared_managed_segment_00", "EXTENT_MAP_RB_TREE");
```

Comment:
It is better to change the suffix from InfiniDB to MCS. Moreover it would be nice if you add the corresponding constants into shmemkeys.h

Reply:
I see, that a bug, thanks.